config/cscs.py

```diff
@@ -990,16 +991,16 @@
                     'datefmt': '%FT%T%:z',
                     'append': True
                 },
-                {
-                    'type': 'graylog',
-                    'address': 'graylog-server:12345',
-                    'level': 'info',
-                    'format': '%(message)s',
-                    'extras': {
-                        'facility': 'reframe',
-                        'data-version': '1.0',
-                    }
-                }
+##                 {
+##                     'type': 'graylog',
+##                     'address': 'graylog-server:12345',
+##                     'level': 'info',
+##                     'format': '%(message)s',
+##                     'extras': {
+##                         'facility': 'reframe',
+##                         'data-version': '1.0',
+##                     }
+##                 }
```

Comment:
You don't need to comment this for testing. ReFrame will simply not initialize the Graylog handler if it cannot connect to it.